suricata.yaml.in

```diff
@@ -1115,6 +1115,11 @@ pcre:
   match-limit: 3500
   match-limit-recursion: 1500
 
+# MQTT topic detection depth
+mqtt:
+  subscribe-topic-match-limit: 100
+  unsubscribe-topic-match-limit: 100
```

Comment:
Does it make sense to have 2 different values ?

Reply:
My idea was that it does not hurt exposing both separately. They are each checked in separate places so it just felt natural to have two config items. They can just be identical, or maybe not. I'd rather make it possible to configure both than limiting what can be configured.

Reply:
These values should be commented out in the default configuration I think and use some hardcoded default.  If we later decide there are better values for the defaults, then users upgrading won't acquire those new defaults automatically which is probably expected unless the user explicityly changed these away from the default values.

Reply:
Good point! We already have hardcoded defaults in the detect code matching the current default in the config, so it should be easy just to comment these out. Will do that in the next PR.